File: represent/bench/java_flat/SVCOMP_0078__SVCOMP__ExSymExeSwitch__ExSymExeSwitch.java
class Main {

  public static boolean f(int x) {
    if (x > 0) return true;
    return test(x);
  }

  public static boolean test(int x) {
    x = x + 1;
    switch (x) {
      case 2:
        System.out.println("branch Foo2");
        return false;
      case 3000:
        System.out.println("branch Foo3000");
        break;
      default:
        System.out.println("default2");
        break;
    }
    return true;
  }
}

File: represent/bench/c_flat/SVCOMP_0078__SVCOMP__ExSymExeSwitch__ExSymExeSwitch.c
int f(int x);
int test(int x);

int f(int x) {

    if (x > 0) return 1;
    return test(x);
}

int test(int x) {

    x = x + 1;
    switch (x) {
      case 2:

        return 0;
      case 3000:

        break;
      default:

        break;
    }
    return 1;
}
